lib/src/features/streammanagement/StreamManagementModule.dart

```diff
@@ -108,11 +107,10 @@ class StreamManagementModule extends Negotiator {
         SMNonza.match(nonzas[0]) &&
         _connection.authenticated) {
       state = NegotiatorState.NEGOTIATING;
-      inNonzaSubscription = _connection.inNonzasStream.listen(parseNonza);
-      if (streamState.isResumeAvailable()) {
-        tryToResumeStream();
-      } else {
-        //sendEnableStreamManagement();
+      enablingStream = false;
+      // Somehow, it listens too many times, if we don't clear or check.
+      if (inNonzaSubscription == null) {
```

Comment:
@slashdigital this also needed to be controlled because it can listen more and more